### ISSUE TITLE
Smart operator selection

### ISSUE THREAD

**Issue by kolayne** [kolayne/anonymous_helpline_chatbot#4]
Do not select operators randomly. Perhaps decide which operators to choose based on:
- Who did the client chat with (and whether he/she liked the conversion)
- ~~How much clients is an operator speaking to set the moment~~ (after #18 each operator communicates with no more than 1 client at a time)
- Whether an operator is online and/or accepted the conversion

**Comment by Beyzary** [comment]
1 and 3 are the best

**Comment by kolayne** [comment]
@Beyzary, please, give me a bit more details on how you want this to work

**Comment by kolayne** [comment]
https://stackoverflow.com/a/38131571/11248508 :(

**Comment by Beyzary** [comment]
Hi, @kolayne! Sorry for such a long delay.

Because it's impossible to get the "last seen online" date from a telegram bot, I believe the best way is to send out a message, asking about their readiness to start a conversation, to all the free operators. The operator, who reacts first, gets the conversation. The prompt is deleted in the chats with the other operators

**Comment by kolayne** [comment]
Thanks, @Beyzary. Let's make sure we understand each other again. Here's a summary of what I'm implementing. Are there any misunderstandings?
https://github.com/kolayne/anonymous_helpline_chatbot/blob/5f668263f1417b37d9d119c61d086e95caaff4c4/logic.py#L94-L96
UPD: I meant "all the free operators" instead of "all the operators" in the first line

**Comment by kolayne** [comment]
@Beyzary?..

**Comment by Beyzary** [comment]
@kolayne, absolutely right! Waiting for the function of communication with a "favourite" operator!..

**Comment by kolayne** [comment]
@Beyzary, I'm pretty sure this should be a separate issue. Will you file it, please?
